src/main/views/subscription-add-list-language.njk

```diff
@@ -0,0 +1,71 @@
+{% extends "template.njk" %}
+{% from "macros/common-components.njk" import submitButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% block pageTitle %}
+    {{ title }}
+{% endblock %}
+
+{% block beforeContent %}
+    {{ govukBackLink({
```

Comment:
Can we not use the goBack() macro defined in common-components here?

Reply:
as discussed, updated.